apple/Handlers/RNPanHandler.m

```diff
@@ -78,6 +86,24 @@ - (void)setMinimumNumberOfTouches:(NSUInteger)minimumNumberOfTouches
 {
   _realMinimumNumberOfTouches = minimumNumberOfTouches;
 }
+
+- (void)tryUpdateStylusData:(UIEvent *)event
+{
+  UITouch *touch = [[event allTouches] anyObject];
+
+  if (touch.type != UITouchTypePencil) {
+    return;
+  }
```

Comment:
Can you use both finger and pencil at the same time?

Reply:
No, you can't